tests/topotests/pim-basic/r1/bgpd.conf

```diff
@@ -0,0 +1,3 @@
+router bgp 65001
+  neighbor 10.0.30.3 remote-as external
+  redistribute connected
```

Comment:
can this GitHub warning be fixed? ![Screenshot from 2020-01-18 19-53-30](https://user-images.githubusercontent.com/3352707/72668129-36c3e000-3a2c-11ea-9ccf-ff579c6c6b6d.png)